index.html

```diff
@@ -61,43 +17,45 @@
     </div>
   </div>
+
   <div class="container">
     <h1>Gunn Programming Homepage</h1>
-    <p>We're also on <a class="link" href="https://github.com/nbuilding">Github</a> and <a class="link" href="https://tinyurl.com/GunnDiscord" target="_blank">Discord</a>.</p>
+    <p>Check us out on <a class="link" href="https://github.com/nbuilding">Github</a> and <a class="link" href="https://tinyurl.com/GunnDiscord" target="_blank">Discord</a>!</p>
   </div>
-  
+
   <div class="container">
     <h2>About us</h2>
     <p>
         We are <a class="link" href="https://gunn.pausd.org">Gunn</a> people who often sit in the N building of
       <a class="link" href="https://tinyurl.com/GunnDiscord" target="_blank">Gunn Discord</a>.
     </p>
-    
+
     <p>
-      We do <b>NOT</b> live under the stairs or in the basement of the N building. Please recheck your sources.
+      <!-- Who is this directed towards? -->
+      We do <b>NOT</b> live under the stairs or in the basement of the N building.
     </p>
   </div>
-  
+
   <div class="container">
     <h2>Members</h2>
     <ul>
-      <li>Tomer Sedan</li>
-      <li>Brandon Chung</li>
-      <li>Stanley Zhong</li>
-      <li>Sean Yen</li>
-      <li>Steven Xue</li>
-      <li>Timothy Herchen</li>
-      <li>Akul Datta</li>
-      <li>Serena Li</li>
-      <li>Alina Li</li>
-      <li>Om Mahesh</li>
-      <li>Roger Fan</li>
-      <li>Alexander Chang-Davidson</li>
-      <li>Kevin Yu</li>
-      <li>Saumya Singhal</li>
-      <li>Matthew Grupenhoff</li>
-      <li>Yu-Ting Chang</li>
-      <li>William Tsin</li>
+      <li><a class="link" href="https://github.com/Sordamente">Tomer Sedan</a></li>
+      <li><a class="link" href="https://github.com/Nichodon">Brandon Chung</a></li>
+      <li><a class="link" href="https://github.com/qpwoeirut">Stanley Zhong</a></li>
+      <li><a class="link" href="https://github.com/SheepTester">Sean Yen</a></li>
+      <li><a class="link" href="https://github.com/heaucques">Steven Xue</a></li>
+      <li><a class="link" href="https://github.com/anematode">Timothy Herchen</a></li>
+      <li><a class="link" href="https://github.com/AkulDatta">Akul Datta</a></li>
+      <li><a class="link" href="https://github.com/acrantel">Serena Li</a></li>
+      <li><a class="link" href="https://github.com/pinkbluesky">Alina Li</a></li>
+      <li><a class="link" href="https://github.com/omahesh">Om Mahesh</a></li>
+      <li><a class="link" href="https://github.com/BubbyBabur">Roger Fan</a></li>
+      <li><a class="link" href="https://github.com/acd124">Alexander Chang-Davidson</a></li>
+      <li><a class="link" href="https://github.com/ky28059">Kevin Yu</a></li>
+      <li><a class="link" href="https://github.com/saumyasinghal747">Saumya Singhal</a></li>
+      <li><a class="link" href="https://github.com/mg27713">Matthew Grupenhoff</a></li>
+      <li><a class="link" href="https://github.com/ytchang05">Yu-Ting Chang</a></li>
+      <li><a class="link" href="">William Tsin</a></li>
```

Comment:
Of the members in the organization, @bucketpotato01 isn't listed, so it's probably him

Reply:
Epic, will add!

Reply:
What if instead of just linking to our githubs we had bio pages that had links to our githubs and homepages?

Reply:
Yeah, I was thinking that perhaps we could show the list of people as cards, using Github profile pictures as an image and perhaps linking to other social media :eyes:

Reply:
Good idea, I'll try adding that as well.